tools/code-owners-parser/custom-tests.yml

```diff
@@ -0,0 +1,6 @@
+steps:
+  - pwsh: |
+      $output = dotnet run --project "tools\code-owners-parser\Azure.Sdk.Tools.RetrieveCodeOwners\Azure.Sdk.Tools.RetrieveCodeOwners.csproj" `
```

Comment:
I thought the plan was to wait to add this testing once we make the script update so that we can add this an potentially more tests to the script itself.